.github/scripts/deploy.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+echo "Deploying application ..."
+
+# Get the current branch
+git pull
+
+# Install npm dependencies
+npm install
+
+# Build the project
+npm run build
+
+# Start the server
+pm2 restart ecosystem.config.js
+
+echo "Application deployed!"
```

Comment:
```suggestion
echo "Application deployed!"

```

add newline at the end of file